### ISSUE TITLE
Failed to load dynamic library

### ISSUE THREAD

**Issue by stevanusdk** [Levi-Lesches/opencv_ffi#17]
Hi Levi,
I am creating a brand new app using the example code in MacOS,
but I got an error saying that :

```
Failed to load dynamic library 
'opencv_ffi.dylib': dlopen(opencv_ffi.dylib, 1): image not found
```

I have tried to change the library dylib path but no luck,
Is the plugin working on MacOS?
Do I need to build the openCV folder?

Flutter Version:
Flutter 3.13.0
ffigen : ^9.0.0 (tried 6.1.2)

thank you for your amazing plugins

Best Regards

**Comment by Levi-Lesches** [comment]
Yes, you do need to build `opencv` with `cmake` in the `src` folder, then get Dart to recognize your `.dylib` files. I personally do not have a Mac device so it's hard for me to test, but if you can get it working I'd be interested in hearing how so I can document it!

**Comment by Levi-Lesches** [comment]
Hi @stevanusdk, I made a change at https://github.com/Levi-Lesches/opencv_ffi/commit/fb721ce518faa62b470c73ae58edfe825a5f9052 according to @tomtom101 in #19. Can you try adding the following to your `pubspec.yaml`, running `flutter pub get` and trying again?

```yaml
dependency_overrides:
  opencv_ffi: 
    git: https://github.com/Levi-Lesches/opencv_ffi.git
```

**Comment by fusionconsulting** [comment]
> Hi Levi, I am creating a brand new app using the example code in MacOS, but I got an error saying that :
> 
> ```
> Failed to load dynamic library 
> 'opencv_ffi.dylib': dlopen(opencv_ffi.dylib, 1): image not found
> ```
> 
> I have tried to change the library dylib path but no luck, Is the plugin working on MacOS? Do I need to build the openCV folder?
> 
> Flutter Version: Flutter 3.13.0 ffigen : ^9.0.0 (tried 6.1.2)
> 
> thank you for your amazing plugins
> 
> Best Regards

Did you get this building on MacOS? I'd be interested to know if it works.
